History.md

```diff
@@ -10,6 +10,11 @@
   * Changed #connected_port to #connected_ports (#2076)
   * `--control` has been removed. Use `--control-url` (#1487)
   * `worker_directory` has been removed. Use `directory`
+  * min_threads now set by environment variables RAILS_MIN_THREADS and MIN_THREADS
```

Comment:
Sorry if I wasn't clear - `/min_threads/puma_min_threads/` and `/max_threads/puma_max_threads` please.